test/test_completers/test_find_bins_in_path.py

```diff
@@ -0,0 +1,48 @@
+from unittest.mock import patch
+import os
+from pypsi.os import find_bins_in_path
+
+
+class TestPathCompleter:
+
+    def setup(self):
+        base_path = os.path.dirname(os.path.abspath(__file__))
+        self.actual_path = os.environ.get('PATH', '')
+
+        self.environ_patch = patch('os.environ')
+        self.environ_mock = self.environ_patch.start()
+
+    def teardown(self):
+        self.environ_mock.stop()
+
+    def test_path_unset(self):
+        # An empty or missing PATH will result in checking `./`, which checks the current repo
+        self.environ_mock.return_value = f"./"
```

Comment:
The CI tests run on pre-Python 3.7. Can you update to remove f-strings and use `%` or `.format()` instead?

Reply:
I just pushed some simplified unit tests that don't use f-strings (the old unit tests didn't actually test anything!). I'll make sure there aren't any other newer features in use.